logger/logger.go

```diff
@@ -1,30 +1,30 @@
 package logger
 
 import (
-	"os"
+  "os"
```

Comment:
Sorry, I think we need to fix spacing on this file. `We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.`

Reply:
Good point. Will update spacing from spaces to tabs.